webrtc.html

```diff
@@ -523,39 +529,45 @@
           </li>
         </ol>
 
-        <p>Once the RTCPeerConnection object has been initialized, for every
-        call to <code>createOffer</code>, <code>setLocalDescription</code>,
-        <code>createAnswer</code>, <code>setRemoteDescription</code>,
-        and <code>addIceCandidate</code>,
-        execute the following steps:</p>
+        <p>An <code><a>RTCPeerConnection</a></code> object has an
+        <dfn>operations queue</dfn>, [[<a>operations</a>]], which enforces that
+        only one operation in the queue is executed at a given time. If
+        subsequent calls are made while the returned promise of a previous call
+        is still unsettled, they are added to the queue and executed when all
+        the previous calls are executed and their promises are settled.</p>
+
+        <p>To <dfn id="enqueue-an-operation">enqueue an operation</dfn>, run the
+        following steps:</p>
 
         <ol>
           <li>
+            <p>Let <var>operation</var> be the operation to be enqueued.</p>
```

Comment:
I'm a bit hesitant to use 'function call' since that somewhat indicates that we're enqueuing a function, when we're actually queueing a part of a function.